### PR TITLE
First Update README.md

### PR DESCRIPTION
**Pull request by JoePehlke**
### 🤔 Reasoning

Update the readme to include installation instructions

### 🚧 Changes

add a header and an installation page

### ✅ PR Checklist

- [x] Using this PR template?
- [x] Linked issue?
- [x] Added feature?
    - [x] Added/updated docs?
    - [x] Added/updated tests?

### CODE REVIEW

**Comment by alckasoc** [comment]
🥳🥳

File: README.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 Discussion agents.
 
+# Windows installation 
+```
+1.First install anaconda https://docs.anaconda.com/free/anaconda/install/windows/ and add to path, or launch anaconda console
```
Comment (alckasoc) [inline README.md:7]: Add a space after the number

Comment (alckasoc) [reply README.md:7]: Also, the link can be formatted with markdown like this [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/). End each step with a period (full sentences).

Comment (alckasoc) [reply README.md:7]: Probably go into detail (alternatively, you can add a link to some resource for how to add stuff to the path) describing how to add to the path.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+2.conda create -n my_env python=3.10.13
```
Comment (alckasoc) [inline README.md:8]: Code can go in the code block and the text can go outside of the code block. Maybe add a statement above the code block to describe the step, the command, and the flags. 

What is `conda create`?
What is `-n`?
What is the `my_env` argument?
What is the `python=3.10.13`?

File: README.md [hunk 3]
```diff
@@ -2,6 +2,15 @@
...
+3.conda activate my_env
+4.pip install pipx
```
Comment (alckasoc) [inline README.md:10]: Maybe also add a link to pipx docs for easy reference.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,15 @@
...
+5.pipx install poetry (make sure to add poetry to path by adding C:\Users\<username\.local\bin to path in advanced system settings)
```
Comment (alckasoc) [inline README.md:11]: Missing ">" sign after "username". Probably want to go into detail about adding the path.

File: README.md [hunk 5]
```diff
@@ -2,6 +2,15 @@
...
+6.In the discussion agents directory - poetry install
```
Comment (alckasoc) [inline README.md:12]: Maybe add specify how a developer can test that their environment is properly set up.

File: README.md [hunk 6]
```diff
@@ -2,6 +2,15 @@
 
 Discussion agents.
 
+# Windows installation
```
Comment (alckasoc) [inline README.md:5]: Developer Setup: Windows

File: README.md [hunk 7]
```diff
@@ -2,6 +2,15 @@
...
 
+# Windows installation 
+```
+1.First install anaconda https://docs.anaconda.com/free/anaconda/install/windows/ and add to path, or launch anaconda console
+2.conda create -n my_env python=3.10.13
+3.conda activate my_env
+4.pip install pipx
+5.pipx install poetry (make sure to add poetry to path by adding C:\Users\<username\.local\bin to path in advanced system settings)
+6.In the discussion agents directory - poetry install
+```
```
Comment (alckasoc) [inline README.md:13]: Newline

File: README.md [hunk 8]
```diff
@@ -1,7 +1,35 @@
 # discussion-agents
 
-Discussion agents.
+Discussion agents. 
 
+# Developer Setup: Windows
+
+1. First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and add to path by going to advanced system settings. Then launch cmd.
+
+Use the following command to create a conda environment named my_env with python version 3.10.13.
```
Comment (alckasoc) [inline README.md:9]:
```suggestion
Use the following command to create a conda environment `discussion-agents` with Python version 3.10.13. Any Python version above 3.9 is viable.
```

File: README.md [hunk 9]
```diff
@@ -1,7 +1,35 @@
...
+```
+2. conda create -n my_env python=3.10.13
```
Comment (alckasoc) [inline README.md:11]:
```suggestion
conda create -n my_env python=3.10.13
```

File: README.md [hunk 10]
```diff
@@ -1,7 +1,35 @@
...
+```
+Now activate the environent.
+```
+3. conda activate my_env
```
Comment (alckasoc) [inline README.md:15]:
```suggestion
conda activate my_env
```

File: README.md [hunk 11]
```diff
@@ -1,7 +1,35 @@
...
+```
+Next install [pipx](https://pipx.pypa.io/stable/docs/) to install [poetry](https://python-poetry.org/docs/).
+```
+4. pip install pipx
+5. pipx install poetry
```
Comment (alckasoc) [inline README.md:20]:
```suggestion
pip install pipx
pipx install poetry
```

File: README.md [hunk 12]
```diff
@@ -1,7 +1,35 @@
...
+
+Use the following command to create a conda environment named my_env with python version 3.10.13.
+```
+2. conda create -n my_env python=3.10.13
+```
+Now activate the environent.
+```
+3. conda activate my_env
+```
+Next install [pipx](https://pipx.pypa.io/stable/docs/) to install [poetry](https://python-poetry.org/docs/).
```
Comment (alckasoc) [inline README.md:17]:
```suggestion
Next, we will install [Poetry](https://python-poetry.org/docs/) using [pipx](https://pipx.pypa.io/stable/docs/).
```

File: README.md (hunk 11 above)
Comment (alckasoc) [inline README.md:20]:
```suggestion
pip install pipx
pipx install poetry
```

File: README.md [hunk 13]
```diff
@@ -1,7 +1,35 @@
...
+```
+4. pip install pipx
+5. pipx install poetry
+```
+Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings.
+
+Then clone the repository and enter the discussion-agents directory.
+``` 
+6. git clone https://github.com/alckasoc/discussion-agents/
```
Comment (alckasoc) [inline README.md:26]:
```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

File: README.md [hunk 14]
```diff
@@ -1,7 +1,35 @@
...
+Now activate the environent.
+```
+3. conda activate my_env
+```
+Next install [pipx](https://pipx.pypa.io/stable/docs/) to install [poetry](https://python-poetry.org/docs/).
+```
+4. pip install pipx
+5. pipx install poetry
+```
+Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings.
```
Comment (alckasoc) [inline README.md:22]:
```suggestion
Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.
```

File: README.md (hunk 13 above)
Comment (alckasoc) [inline README.md:26]:
```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

File: README.md [hunk 15]
```diff
@@ -1,7 +1,35 @@
...
+
+Then clone the repository and enter the discussion-agents directory.
+``` 
+6. git clone https://github.com/alckasoc/discussion-agents/
+
+```
+Finally install all of the packages.
+```
+7. poetry install
```
Comment (alckasoc) [inline README.md:31]:
```suggestion
poetry install
```

File: README.md [hunk 16]
```diff
@@ -1,7 +1,35 @@
 # discussion-agents
 
-Discussion agents.
+Discussion agents. 
 
+# Developer Setup: Windows
```
Comment (alckasoc) [inline README.md:5]:
```suggestion
# Developer Setup
```

File: README.md [hunk 17]
```diff
@@ -1,6 +1,35 @@
...
+
+Use the following command to create a conda environment `discussion-agents` with Python version 3.10.13. Any Python version above 3.9 is viable.
+```
+conda create -n discussion-agents python=3.10.13
+```
+Now activate the environent.
+```
+conda activate discussion-agents
+```
+Next, we will install [Poetry](https://python-poetry.org/docs/) using [pipx](https://pipx.pypa.io/stable/docs/)
```
Comment (alckasoc) [inline README.md:17]:
```suggestion
Next, we will install [Poetry](https://python-poetry.org/docs/) using [pipx](https://pipx.pypa.io/stable/docs/).
```

File: README.md [hunk 18]
```diff
@@ -1,6 +1,35 @@
...
+```
+pip install pipx
+pipx install poetry
+```
+Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.
```
Comment (alckasoc) [inline README.md:22]:
```suggestion
Make sure to add poetry to path by adding `C:\Users\<username>\.local\bin` to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.
```

File: README.md [hunk 19]
```diff
@@ -1,6 +1,35 @@
...
+
+Then clone the repository and enter the discussion-agents directory.
+``` 
+git clone https://github.com/alckasoc/discussion-agents/
+
```
Comment (alckasoc) [inline README.md:27]:
```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

File: README.md [hunk 20]
```diff
@@ -1,6 +1,35 @@
...
+```
+pip install pipx
+pipx install poetry
+```
+Make sure to add poetry to path by adding `C:\Users\<username>\.local\bin` to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.
+
+Then clone the repository and enter the discussion-agents directory.
+``` 
+git clone https://github.com/alckasoc/discussion-agents/
+
```
Comment (alckasoc) [inline README.md:27]:
```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

**Review by alckasoc — APPROVED** [review]

**Review by saishridhar — APPROVED** [review]

**Pull request merged**